tests/integration/analytics/test_analytics_collectors.py

```diff
@@ -575,31 +624,73 @@ def test_event_streams_table_by_activation_collector(
         with open(
             os.path.join(tmpdir, "event_streams_by_activation_table.csv")
         ) as f:
-            reader = csv.reader(f)
+            assert_event_streams_data(
+                default_event_streams, [default_activation, new_activation], f
+            )
 
-            header = next(reader)
-            lines = list(reader)
 
-            assert header == [
-                "name",
-                "event_stream_type",
-                "eda_credential_id",
-                "events_received",
-                "last_event_received_at",
-                "organization_id",
-                "event_stream_id",
-                "activation_id",
-            ]
-            assert len(lines) == 2
-            assert lines[0][0] == default_event_streams[0].name
-            assert lines[0][1] == default_event_streams[0].event_stream_type
-            assert lines[0][6] == str(default_event_streams[0].id)
-            assert lines[1][0] == default_event_streams[1].name
-            assert lines[1][1] == default_event_streams[1].event_stream_type
-            assert lines[1][6] == str(default_event_streams[1].id)
-            assert sorted([lines[0][7], lines[1][7]]) == sorted(
-                [str(default_activation.id), str(new_activation.id)]
+@pytest.mark.django_db
+def test_event_streams_table_by_running_activations_collector(
+    activations_with_different_status: list[models.Activation],
+    default_event_streams: list[models.EventStream],
+):
+    activations_with_different_status[0].event_streams.add(
+        default_event_streams[0]
+    )
+    activations_with_different_status[1].event_streams.add(
+        default_event_streams[1]
```

Comment:
Can you add a case that an event stream is used by multiple activations? Can it be deduplicated?

Reply:
Yes, even streams used by different activations are treated as other records in the collection table.